src/Installer/Elastic.Installer.Msi/_Generated/elasticsearch.wxs

```diff
@@ -537,63 +586,838 @@
+
+                          <Component Id="Component.libMlModel.so" Guid="daaa2cba-a1ed-4f29-afbf-5478793349da" Win64="yes">
+                            <File Id="libMlModel.so" Source="..\..\..\build\in\elasticsearch-6.3.0\modules\x-pack\x-pack-ml\platform\linux-x86_64\lib\libMlModel.so" />
+                          </Component>
+
+                          <Component Id="Component.libstdc__.so.6_" Guid="daaa2cba-a1ed-4f29-afbf-5478966bd1b5" Win64="yes">
+                            <File Id="libstdc__.so.6_" Source="..\..\..\build\in\elasticsearch-6.3.0\modules\x-pack\x-pack-ml\platform\linux-x86_64\lib\libstdc++.so.6" />
+                          </Component>
+
+                          <Component Id="Component.libxml2.so.2_" Guid="daaa2cba-a1ed-4f29-afbf-5478d448d0f9" Win64="yes">
+                            <File Id="libxml2.so.2_" Source="..\..\..\build\in\elasticsearch-6.3.0\modules\x-pack\x-pack-ml\platform\linux-x86_64\lib\libxml2.so.2" />
+                          </Component>
+
+                        </Directory>
+                      </Directory>
+
+                      <Directory Id="INSTALLDIR.modules.x_pack.x_pack_ml.platform.windows_x86_64" Name="windows-x86_64">
+                        <Directory Id="INSTALLDIR.modules.x_pack.x_pack_ml.platform.windows_x86_64.bin" Name="bin">
+
+                          <Component Id="Component.INSTALLDIR.modules.x_pack.x_pack_ml.platform.windows_x86_64.bin" Guid="daaa2cba-a1ed-4f29-afbf-547876ee0d55" Win64="yes">
+                            <RemoveFile Id="INSTALLDIR.modules.x_pack.x_pack_ml.platform.windows_x86_64.bin.all" Name="*" On="both" />
+                            <RemoveFolder Id="INSTALLDIR.modules.x_pack.x_pack_ml.platform.windows_x86_64.bin.dir" On="both" />
+                          </Component>
+
+                          <Component Id="Component.autoconfig.exe" Guid="daaa2cba-a1ed-4f29-afbf-54786173363d" Win64="yes">
+                            <File Id="autoconfig.exe" Source="..\..\..\build\in\elasticsearch-6.3.0\modules\x-pack\x-pack-ml\platform\windows-x86_64\bin\autoconfig.exe" />
+                          </Component>
+
+                          <Component Id="Component.autodetect.exe" Guid="daaa2cba-a1ed-4f29-afbf-547808510c94" Win64="yes">
+                            <File Id="autodetect.exe" Source="..\..\..\build\in\elasticsearch-6.3.0\modules\x-pack\x-pack-ml\platform\windows-x86_64\bin\autodetect.exe" />
+                          </Component>
+
+                          <Component Guid="daaa2cba-a1ed-4f29-afbf-5478fa02a0ad" Win64="yes" Id="Component.INSTALLDIR_modules_x_pack_x_pack_ml_platform_windows_x86_64_bin.boost_chrono_vc141_mt_1_65_1.dll">
+                            <File Source="..\..\..\build\in\elasticsearch-6.3.0\modules\x-pack\x-pack-ml\platform\windows-x86_64\bin\boost_chrono-vc141-mt-1_65_1.dll" Id="INSTALLDIR_modules_x_pack_x_pack_ml_platform_windows_x86_64_bin.boost_chrono_vc141_mt_1_65_1.dll" />
```

Comment:
The `141` in all these Boost library filenames will be `120` in 6.x, as we're still using VS2013 to compile 6.x.  The `141` corresponds to VS2017, which we'll use to build 7.x.

Similarly the MS runtime libraries will be different between the two branches.  For example `concrt140.dll` is a VS2017 file and 6.x will instead ship `msvcr120.dll`.

I guess all this will be fine if you regenerate this file from an ES 6.3.0 build when backporting to the 6.x branch.  It just looks a little weird as it stands now to someone who knows about the ML C++ because all the paths contain `build\in\elasticsearch-6.3.0` but the files are from 7.0.0.  So I just wanted to make sure you're aware that 6.x and master have different ML C++ files to be installed.

Reply:
Absolutely @droberts195!, This was build using a fake 6.3.0 from the xpack master branch, because the 6x branch wasnt merged yet. We'll update accordingly before pulling. Thanks for lending us your eye!